spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Train.scala

```diff
@@ -118,6 +120,19 @@ object TrainInceptionV1 {
       if (param.gradientL2NormThreshold.isDefined) {
         optimizer.setGradientClippingByl2Norm(param.gradientL2NormThreshold.get.toFloat)
       }
+      
+      if (param.visualization) {
+        val logdir = "bigdlSummary"
```

Comment:
As mentioned above, can you let user input the logdir instead of hard code one in the code